### PR TITLE
Summary View Unit Tests

### PR DESCRIPTION
**Pull request by DiegoManzanarezx**
This pull request includes:

- Unit tests for Summary View.

### CODE REVIEW

File: test-runner-frontend/package-lock.json [hunk 1]
```diff
@@ -3997,9 +3997,9 @@
...
     "@types/react-dom": {
-      "version": "18.0.5",
-      "resolved": "https://artifactory.twitter.biz/artifactory/api/npm/js-virtual/@types/react-dom/-/react-dom-18.0.5.tgz",
-      "integrity": "sha512-OWPWTUrY/NIrjsAPkAk1wW9LZeIjSvkXRhclsFO8CZcZGCOg2G0YZy4ft+rOyYxy8B7ui5iZzi9OkDebZ7/QSA==",
+      "version": "18.0.6",
+      "resolved": "https://artifactory.twitter.biz/artifactory/api/npm/js-virtual/@types/react-dom/-/react-dom-18.0.6.tgz",
+      "integrity": "sha512-/5OFZgfIPSwy+YuIBP/FgJnQnsxhZhjjrnxudMddeblOouIodEQ75X14Rr4wGSG/bknL+Omy9iWlLo1u/9GzAA==",
       "dev": true,
       "requires": {
         "@types/react": "*"
```
Comment (DanElliottPalmer) [inline test-runner-frontend/package-lock.json:4005]: can we please remove the changes from this file? you only usually want to commit package-lock.json files when you update package.json

File: test-runner-frontend/src/Views/Dashboard.test.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  test('Check buttons rendering and functionality', async () => {
+    const { debug, getByRole } = render(<Dashboard/>);
+    const CompareButton = getByRole('button', { name: /Compare/i });
+    expect(CompareButton).toBeDefined();
+    await userEvent.click(CompareButton);
+    const CancelButton = getByRole('button', { name: /Cancel/i });
+    expect(CancelButton).toBeDefined();
+    await userEvent.click(CancelButton);
+  });
+  */
```
Comment (DanElliottPalmer) [inline test-runner-frontend/src/Views/Dashboard.test.js:16]: please remove any comments that aren't needed anymore

**Review by DanElliottPalmer — APPROVED** [review]
just two comments then merge away 👍

File: test-runner-frontend/src/Views/Dashboard.test.js (hunk 2 above)
Comment (DiegoManzanarezx) [reply test-runner-frontend/src/Views/Dashboard.test.js]: Done

File: test-runner-frontend/package-lock.json (hunk 1 above)
Comment (DiegoManzanarezx) [reply test-runner-frontend/package-lock.json]: Done

**Pull request merged**